packages/paste-website/src/pages/components/truncate/index.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: Truncate
+package: '@twilio-paste/truncate'
+description: The Truncate component can be used to shorten a string of text that overflows a defined area.
+slug: /components/truncate/
+---
+
+import {graphql} from 'gatsby';
+import {Alert} from '@twilio-paste/alert';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Heading} from '@twilio-paste/heading';
+import {Text} from '@twilio-paste/text';
+import {Truncate} from '@twilio-paste/truncate';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import Changelog from '@twilio-paste/truncate/CHANGELOG.md';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/truncate"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/truncate/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/truncate"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-truncate--default"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Truncate
+
+Truncation, or shortening, is typically used for text or links that exceed the size of their container.
+The Truncate component does all of the heavy lifting for you. You'll need to wrap the long
+text with Truncate, and wrap the Truncate component with an element with a set width. This will cut off the text based on
+the parent container's width, and append an ellipsis (`...`) at the end.
+
+Good use cases for truncation include:
+
+- Breadcrumbs
+- Long URL links
+
+Truncation should <strong>NOT</strong> be used on page headers, titles, labels, error messages, validation messages,
+notifications, or SIDs.
```

Comment:
To prevent the obvious question coming in you need to clearly state that truncation can only happen on single lines of text and will never work on multiple lines. Multiline needs to be controlled with JavaScript and string trimming

Reply:
Good call. Added.